apps/system/js/attention_window_manager.js

```diff
@@ -45,6 +45,14 @@
       return this._topMostWindow;
     },
 
+    showCallscreenWindow: function attwm_getCallscreenWindow() {
+      var callscreenWindow = this._instances.find(function(aw) {
+        return aw.isCallscreenWindow;
+      });
+
+      callscreenWindow && callscreenWindow.requestOpen();
+    },
```

Comment:
worth testing since it'll make the code much cleaner, but never mind otherwise.

I _think_ that the window manager is supposed to do the right thing if the app calls `window.open` with the same params again (ie. expand the attention window instead of creating a new one).